prov/tcp/src/tcpx_conn_mgr.c

```diff
@@ -130,7 +130,10 @@ static int tx_cm_data(SOCKET fd, uint8_t type, struct tcpx_cm_context *cm_ctx)
 	return FI_SUCCESS;
 }
 
-static int tcpx_ep_enable(struct tcpx_ep *ep)
+static int tcpx_ep_enable(struct tcpx_ep *ep,
+				   struct fi_eq_cm_entry *cm_entry,
+				   size_t cm_entry_sz)
```

Comment:
nit: can you fix the alignment here.

Also, can you expand the commit message.  The reason for the TODO was to delay reporting that the EP was connected until after it had been fully setup